src/events/event/EventPage.js

```diff
@@ -111,19 +112,18 @@ const EventPage = ({
           },
           {
             property: 'og:latitude',
-            content: location.lat,
+            content: location && location.lat,
```

Comment:
Doesnt this just generate a meta tag without any content? does helmet handle this automatically, or should we probably exclude the entire object from the array if location isnt defined

Reply:
I thought it handled it automatically, but apparently not. I've updated ✅